### ISSUE TITLE
case.st_archive --test  doesn't work

### ISSUE THREAD

**Issue by jgfouca** [ESMCI/cime#2522]
```
% ./create_test SMS.f45_g37_rx1.A --no-build
...
% cd $case
% ./case.st_archive --test
...
ERROR: Failed to move file mali.hist.am.globalStats.1976-01-01.nc to archive
```

**Comment by jedwards4b** [comment]
This is fixed by #2519

**Comment by jgfouca** [comment]
@jedwards4b , thx
